bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -276,11 +333,9 @@ impl<'a> CircuitInputStateRef<'a> {
         value: Word,
         value_prev: Word,
     ) -> Result<(), Error> {
-        self.push_op(
-            step,
-            RW::READ,
-            AccountOp::new(address, field, value, value_prev),
-        );
+        let op = AccountOp::new(address, field, value, value_prev);
+        self.check_update_sdb_account(RW::READ, &op);
+        self.push_op(step, RW::READ, op);
```

Comment:
what about put the logic of check_update_sdb_account inside `push_op`? So in the future developers never omit this check.

```
fn push_op(rw, op, ...):
    if op.type == Account:
       check_update_sdb_account(rw, op)
    ...

```

Reply:
Good idea! I've applied it in a new commit, thanks!